.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.4.2
+2.4.2
```

Comment:
Why is this change needed? It worked fine before on both CI and here locally.

Reply:
hey @djbe, my rbenv failed to find this version, but could find `2.4.2`. it wasn't listed on rbenv list of available versions. don't really know why, but I normally see ruby version declared with only the desired version, without the `ruby-`. this is a problem? want me to revert that?

Reply:
Funny, I also use `rbenv`. I remember spending quite a while getting it to work both locally as on CI. The change without `ruby-` works for me locally, so that doesn't seem to be an issue (anymore). If this works for everyone, then I definitely won't complain 🤷‍♂️ 

Note: what might help is manually specifying what version to install with `rbenv install 2.4.2`.